$NetBSD: patch-Makefile,v 1.2 2024/05/07 10:13:14 nia Exp $

Honour pkgsrc CFLAGS/LDFLAGS.
Use sdl2-config --libs to fix cocoa linking on Darwin.

--- Makefile.orig	2021-11-24 14:25:50.000000000 +0000
+++ Makefile
@@ -4,18 +4,18 @@ SDL_VER?=2
 
 ifeq ($(SDL_VER),2)
 	SDL_CONFIG=sdl2-config
-	SDL_LD=-lSDL2 -lSDL2_mixer
+	SDL_LD=$$(sdl2-config --libs) -lSDL2_mixer
 else
 ifeq ($(SDL_VER),1)
 	SDL_CONFIG=sdl-config
-	SDL_LD=-lSDL -lSDL_mixer
+	SDL_LD=$$(sdl-config --libs) -lSDL_mixer
 else
 	SDL_CONFIG=$(error "invalid SDL version '$(SDL_VER)'. possible values are '1' and '2'")
 endif
 endif
 
-CFLAGS=-Wall -g -O2 `$(SDL_CONFIG) --cflags`
-LDFLAGS=$(SDL_LD)
+CFLAGS+=-std=c99 -Wall `$(SDL_CONFIG) --cflags`
+LDFLAGS+=$(SDL_LD)
 CELESTE_CC=$(CC)
 
 ifneq ($(USE_FIXEDP),)
